pkg/metrics/query.go

```diff
@@ -38,12 +39,11 @@ func Query(
 	// Copy URL to avoid modifying the passed value.
 	u := new(url.URL)
 	*u = *endpoint
-	u.Path = ""
```

Comment:
not really sure why we were setting `u.Path = ""` in the first place? just curious:)

Reply:
Looks like it was cherry-picked from https://github.com/observatorium/up/pull/33 - see my comment; I guess we should first resolve it in that PR.

Reply:
Updated #33 to make metrics read URL paths for `query` and `query_range`